sdk/keyvault/test-resources.json

```diff
@@ -9,45 +9,44 @@
                 "description": "The base resource name."
             }
         },
+        "keyName": {
+            "type": "string",
+            "defaultValue": "testKey",
+            "metadata": {
+                "description": "The initial key in the keys."
+            }
+        },
         "tenantId": {
             "type": "string",
             "defaultValue": "72f988bf-86f1-41af-91ab-2d7cd011db47",
             "metadata": {
                 "description": "The tenant ID to which the application and resources belong."
             }
         },
-        "testApplicationOid": {
+        "testApplicationId": {
             "type": "string",
-            "defaultValue": "b3653439-8136-4cd5-aac3-2a9460871ca6",
             "metadata": {
-                "description": "The client OID to grant access to test resources."
+                "description": "The application client ID used to run tests."
             }
         },
-        "location": {
+        "testApplicationSecret": {
             "type": "string",
-            "defaultValue": "[resourceGroup().location]",
             "metadata": {
-                "description": "The location of the resource. By default, this is the same as the resource group."
+                "description": "The application client secret used to run tests."
             }
         },
-        "hsmLocation": {
+        "testApplicationOid": {
             "type": "string",
-            "defaultValue": "southcentralus",
-            "allowedValues": [
-                "eastus2",
-                "southcentralus",
-                "northeurope",
-                "westeurope"
-            ],
+            "defaultValue": "b3653439-8136-4cd5-aac3-2a9460871ca6",
```

Comment:
Should this be generated and passed in by the tests such that multiple tests can run in parallel without stomping on each other.